.isort.cfg

```diff
@@ -1,5 +0,0 @@
-[settings]
```

Comment:
Can you explain what the purpose of the file was before this change and why it has been removed?